slick-testkit/src/test/scala/slick/test/codegen/GeneratedCodeTest.scala

```diff
@@ -85,9 +85,12 @@ object GeneratedCodeTest {
     import CG3._
     import profile.api._
     val s = Supplier(49, "Superior Coffee", "1 Party Place", "Mendocino", "CA", "95460")
+    val ts = java.sql.Timestamp.valueOf("2016-08-30 09:14:56.909")
     DBIO.seq(
       schema.create,
       Suppliers += s,
+      Timestamp += TimestampRow( ts ),
+      Timestamp.result.head.map{ r => assertEquals(ts, r.ts) },
```

Comment:
I guess one thing that would be nice here if we don't provide any values, let all values be generated from teh default but then read them back out and check they are correct.